content/en/docs/tasks/observability/logs/access-log/index.md

```diff
@@ -21,8 +21,12 @@ The standard output of Envoy's containers can then be printed by the `kubectl lo
 
 Edit the `istio` configuration map:
 
+{{< tip >}}
+In the example below, replace `demo` with the name of the profile you used when you installed Istio.
```

Comment:
if you follow this tip, and the profile lacks kiali (or prometheus iirc from this review) - some of the tutorial will be nonfunnctional.

Reply:
I added this tip because when testing, I forgot to add the demo profile flag and it removed things and broke the tutorial.